Wikipedia/Code/NotificationsCenterCellViewModel.swift

```diff
@@ -47,6 +47,24 @@ final class NotificationsCenterCellViewModel {
         
     }
     
+    func accessibilityText() -> NSAttributedString? {
```

Comment:
This is super minor, but is there any reason this should be a function instead of a computed var? 
```suggestion
    var accessibilityText: NSAttributedString? {
```

Reply:
Great catch - I guess I initially thought it would require more complex operations to build this string